lib/matplotlib/ticker.py

```diff
@@ -2380,21 +2383,26 @@ def nonsingular(self, vmin, vmax):
             return 1, 10  # initial range, no data plotted yet
 
         if vmin > vmax:
+            swap = True
```

Comment:
I'd rather
~~~
swap = False
if vmin > vmax:
    swap = True
    vmin, vmax = vmax, vmin
~~~

Which is shorter and IMHO better understandable.

Also, maybe `vlims_swapped` might be more expressive.

Reply:
sure, fixed